### PR TITLE
MDTZ-1124: prevent duplicate urls in issue properties

### PR DESCRIPTION
**Pull request by achandra-atlas**
Compares designs using `fileKey` and `nodeId` before writing to / deleting from issue properties to avoid adding duplicate designs 

@akostevich-atlassian looks like we might have some conflicts, so happy to wait till after your PR is merged

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -6,7 +6,7 @@
 APP_URL=
 # A unique key to identify the app.
 # See for more detail: https://developer.atlassian.com/cloud/jira/platform/connect-app-descriptor/
-APP_KEY=com.figma.jira-addon-dev
```
Comment (achandra-atlas) [inline .env.example:9]: to align this with the value we use on the frontend

Comment (akostevich-atlassian) [reply .env.example:9]: **praise:** Nice catch, Aditi!

File: src/domain/entities/testing/mocks.ts [hunk 2]
```diff
@@ -149,8 +149,18 @@ export const generateConnectInstallation = ({
...
-	url = `https://www.figma.com/file/UcmoEBi9SyNOX3SNhXqShY/${displayName}?type=design&node-id=0-1&mode=design`,
-	liveEmbedUrl = `https://www.figma.com/file/UcmoEBi9SyNOX3SNhXqShY/${displayName}?type=design&node-id=0-1&mode=design`,
+	url = generateFigmaDesignUrl({
+		fileKey: getFileKeyFromId(id),
+		nodeId: getNodeIdFromId(id),
+		fileName: displayName,
+		mode: 'design',
+	}),
+	liveEmbedUrl = generateFigmaDesignUrl({
+		fileKey: getFileKeyFromId(id),
```
Comment (akostevich-atlassian) [inline src/domain/entities/testing/mocks.ts:159]: **nitpick** Consider using the existing API to get Figma `fileKey` and `nodeId` from `AtlassianDesign.id`.

```ts
url = generateFigmaDesignUrl({
		fileKey: FigmaDesignIdentifier.fromAtlassianDesignId(id).fileKey,
		nodeId: FigmaDesignIdentifier.fromAtlassianDesignId(id).nodeId,
    // ...
})
```

As a result, we could remove `getNodeIdFromId` and `getFileKeyFromId` from this file :)

**Review by stannnous — APPROVED** [review]

**Review by akostevich-atlassian — CHANGES_REQUESTED** [review]

**Comment by akostevich-atlassian** [comment]
@achandra-atlas , I've got the intention of the change, and will be able to merge with my PR, so please, go ahead with merging :) 

I have only one question about updating `attached-design-url-v2` -- let's discuss this after the standup.

File: src/domain/entities/testing/mocks.ts (hunk 2 above)
Comment (achandra-atlas) [reply src/domain/entities/testing/mocks.ts:159]: ah nice!

**Review by akostevich-atlassian — APPROVED** [review]

**Pull request merged**